index.html

```diff
@@ -15,31 +15,294 @@
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="aboutme-button">See Project</button>
+                    </div>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+
+        </section>
+        <section class="About-me">
+            <div class="aboutme-text">
+                <div class="about-title">
+                    <h1>About me</h1>
+                </div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, </span>
+                    <br /> don’t hestiate to contact me.
+                </p>
+                <button class="aboutme-button">Get My Resume</button>
+            </div>
+            <div class="aboutme-card">
```

Comment:
- [ ] Please kindly ensure that your technologies section is aligned to the center as on the Figma design.

| Your Project | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/219885299-f4dff867-119f-4fca-9ca5-fb083d7d9b48.png) | ![image](https://user-images.githubusercontent.com/52934128/219885325-7d4ae16c-8a9e-4798-8a9d-1916c48e7784.png)  |

Reply:
This 👇🏽 is still not centered as requested by the previous reviewer, look to address that. 🤓 
![Screenshot 2023-02-18 at 17 19 25](https://user-images.githubusercontent.com/45739963/219901793-b3ad59ca-66df-4f3d-8d95-50e859d73b65.png)